src/util/types.ts

```diff
@@ -1733,7 +1733,8 @@ export interface AriaLabelOption {
         separator?: {
             middle?: string;
             end?: string;
-        }
+        },
+        columnsToExclude?: number[]
```

Comment:
I see other places in the code use `excludeSeriesId`, so perhaps here we should use `excludeDataId` instead.